routing/DELEGATED_CONTENT_ROUTING_HTTP.md

```diff
@@ -148,6 +189,60 @@ Specification: [ipfs/specs/BITSWAP.md](https://github.com/ipfs/specs/blob/main/B
 
 The server should respect a passed `transport` query parameter by filtering against the `Addrs` list.
 
+#### Bitswap Write Provider Records
+
+```json
+{
+    "Protocol": "transport-bitswap",
+    "Schema": "bitswap",
+    "Signature": "<signature>",
+    "Payload": "<payload>"
```

Comment:
:thought_balloon:  this creates one `bitswap` schema for `GET` and the other `bitswap` for `PUT`. Both under the same name, but with different fields. Bit nasty and error prone.

@masih is this ossified, or is there still time to fix this and use different schema for PUTs?

```suggestion
    "Schema": "bitswap-announce",
    "Signature": "<signature>",
    "Payload": "<payload>"
```

Reply:
There is still time. Changing thanks